pkg/monitor/monitor.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/liveconfig"
 )
 
+var NSGMonitorFrequency = 10 * time.Minute
```

Comment:
Since this is specific to NSG monitor, maybe we can put this in `pkg/monitor/azure/nsg/nsg.go`?

Reply:
I think this is more like a config for the monitor than something specific to the nsg monitoring package.